streetwhisperapp.py

```diff
@@ -173,7 +245,16 @@ def questions_ui(diarize_model):
     rprint("[blue]=============================[blue]")
     # Destination Folder
     rprint(f"[bold]Enter the absolute path to your destination folder:[bold]")
-    destination_selection = input()
+
+    # Check if dest file path is a valid path within system that CLI is running from
+    while True:
+        destination_selection = input()
+        destination_selection = destination_selection.strip()  # remove leading and trailing whitespace
+        is_valid_dest_path = validate_path(destination_selection, False)
+        if is_valid_dest_path:
+            break
+        else:
+            print("You entered an invalid destination path. Please try again")
```

Comment:
Similar to what was mentioned previously for invalid audio path's statement, maybe you can include a colon at the end of the statement  (So it becomes "You entered an invalid destination path. Please try again:") so that users know that they can enter a new destination path again